src/main/java/thaumicboots/api/serverfiles/PacketJumpToggleAck.java

```diff
@@ -0,0 +1,43 @@
+package thaumicboots.api.serverfiles;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.item.ItemStack;
+
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
+import cpw.mods.fml.common.network.simpleimpl.IMessageHandler;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import io.netty.buffer.ByteBuf;
+import thaumicboots.api.ItemBoots;
+import thaumicboots.main.utils.compat.GTNHLibHelper;
+
+public class PacketJumpToggleAck implements IMessage, IMessageHandler<PacketJumpToggleAck, IMessage> {
+
+    public double state;
+
+    @Override
+    public void fromBytes(ByteBuf byteBuf) {
+        state = byteBuf.readDouble();
+    }
+
+    @Override
+    public void toBytes(ByteBuf byteBuf) {
+        byteBuf.writeDouble(state);
+    }
+
+    @SideOnly(Side.CLIENT)
```

Comment:
`@SideOnly` is not needed here

Reply:
trust me, ***it does***

otherwise it'll crash the server.